02/1_goto_statement/main.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+
+int bad_goto_statement_example(void)
+{
+    int x = 5;
+    goto begin;
+	
+    end:
+        return 0;
+
+    begin:
+        while (1) {
+            --x;
+
+            if (x == 0) {
+                goto end;
+            }
+        }		
+}
+
+int good_goto_statement_example(void)

```

Comment:
This is not a good example of a goto actually. Change this function to have 2 nested loop. There should be an if statement inside the nested loop that returns something from a condition. Else break out of the nested loop using a goto